### PR TITLE
(feature) Add GTM tracking

### PR DESCRIPTION
**Pull request by alexstotsky**
Task: https://app.asana.com/0/1163495710802945/1206077419367370/f

#### Description:
- [x] Implements `Google Tag Manager` support for the `Reports` and events tracking with the following configuration:

```
Send click events for all buttons, nav items & tabs, format example

dataLayer.push({
  event: 'click', // same for all clicks
  label: 'Trades' // text of the button
  area_source: 'Tab' // "Tab" or "Navigation", in case of normal button omit this property
})
```
---
#### Depends on: 
- [bfx-report-ui #761](https://github.com/bitfinexcom/bfx-report-ui/pull/761)

### CODE REVIEW

**Review by elvis-poggi — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@blueprintjs/select": "^3.8.0",
     "@blueprintjs/table": "3.5.0",
     "@blueprintjs/timezone": "3.10.1",
+    "bfxuilib": "git+https://github.com/bitfinexcom/bfxuilib.git",
```
Comment (ezewer) [inline package.json:26]: This is a private repo, wont be able to be added to a public one.

**Review by ezewer — CHANGES_REQUESTED** [review]
Stopped review here as this is blocking.

File: package.json (hunk 1 above)
Comment (alexstotsky) [reply package.json]: Reworked without it.

File: package.json [hunk 2]
```diff
@@ -23,13 +23,15 @@
     "@blueprintjs/select": "^3.8.0",
     "@blueprintjs/table": "3.5.0",
     "@blueprintjs/timezone": "3.10.1",
+    "axios": "1.6.7",
```
Comment (ezewer) [inline package.json:26]: Can we use https://github.com/bitfinexcom/bfx-facs-http please?

**Review by ezewer — CHANGES_REQUESTED** [review]
Small request, rest looks good!

File: package.json (hunk 2 above)
Comment (alexstotsky) [reply package.json:26]: Will update to it a bit later, due to some compatibility issues that are currently found while using it in the UI

Comment (ezewer) [reply package.json:26]: Ok, makes sense.

**Review by ezewer — APPROVED** [review]
looks good to me!

**Pull request merged**